spec/negotiation.html

```diff
@@ -426,8 +426,10 @@ <h1>
       <emu-alg>
         1. If _value_ is *undefined*, return _fallback_.
         1. Set _value_ to ? ToNumber(_value_).
-        1. If _value_ is not finite or ℝ(_value_) &lt; _minimum_ or ℝ(_value_) &gt; _maximum_, throw a *RangeError* exception.
-        1. Return floor(ℝ(_value_)).
+        1. If _value_ is not finite or _value_ is *NaN*, throw a *RangeError* exception.
```

Comment:
```suggestion
        1. If _value_ is not finite, throw a *RangeError* exception.
```

NaN is not finite. See https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#finite.